api/routes/v1.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var express = require('express');
+
+function initRouter(backend) {
+  var router = express.Router();
+
+  function mockResponse(req, res, next) {
+    res.send('This is a mocked response. Backed service is: ' + backend.status);
+  }
+
+  router.get('/blocks', mockResponse);
+  router.get('/block/:blockHash', mockResponse);
```

Comment:
I was planning to heavily refactor the route handling, so I wouldn't mind merging this like this.
(my plan was to add middleware that handles each level of the path)